docs/source/getting-started/installation.rst

```diff
@@ -69,7 +69,7 @@ You will need:
 
 -  `Vue.js <https://vuejs.org/>`__
 -  `vue-cli <https://cli.vuejs.org/>`__
--  `node <https://nodejs.org/en/>`__
+-  `node <https://nodejs.org/en/>`__ (node 8 recommended. Versions greater than node 8 aren't supported)
```

Comment:
Hi @mHash1m, thanks for opening this pull request. I'm unable to reproduce any build-breaking errors when running `make install` with `node` v12.12.0 and `npm` v6.11.3, using macOS `10.14.6.` While I do see `canvas` errors, they don't actually cause anything to break, as `canvas` is an optional dependency as noted in the frontend install logs:
```
> canvas@2.6.1 install /Users/carterlandis/Projects/Work/augur/frontend/node_modules/canvas
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download
[canvas] Success: "/Users/carterlandis/Projects/Work/augur/frontend/node_modules/canvas/build/Release/canvas.node" is installed via remote
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: canvas-prebuilt@1.6.11 (node_modules/canvas-prebuilt):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: canvas-prebuilt@1.6.11 install: `node-pre-gyp install`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: canvas@1.6.13 (node_modules/vue-vega/node_modules/canvas):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: canvas@1.6.13 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
```
Even though I can't reproduce the error, it's still possible it's an Ubuntu specific error, in which case can you please provide the logs files denoting your specific errors of and screenshots of the frontend failing when you try to access it in the broswer since I can't reproduce the error myself?

Reply:
Hi @ccarterlandis , 

I previously opened an [issue](https://github.com/chaoss/augur/issues/551#issuecomment-588261910) about this and also provided a link to the terminal installation logs.
Please take a look and let me know if this is sufficient information. 
If not, I'll try to recreate the issue.

Thanks for reviewing the request.

Reply:
You can see in the [logs](https://gist.github.com/mHash1m/b9f112ec039d137e728524fe33194248) that installing canvas and canvas-prebuilt generates errors.
I found the following at [npmjs site](https://www.npmjs.com/package/canvas-prebuilt).

![Screenshot from 2020-02-24 20-28-28](https://user-images.githubusercontent.com/58783659/75165638-8345be00-5744-11ea-991f-f59164fe1916.png)